packages/core/CHANGELOG.md

```diff
@@ -5,6 +5,53 @@ All notable changes to this project will be documented in this file.
 The file is maintained by [Release Please](https://github.com/googleapis/release-please) based on [Conventional Commits](https://www.conventionalcommits.org) specification,
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.15](https://github.com/waku-org/js-waku/compare/core-v0.0.14...core-v0.0.15) (2023-03-31)
```

Comment:
@fryorcraken because of prev manual bump `release-please` got confused. 
I think it's ok to go and then clean-up changelog, wdyt?

Reply:
Fixing it by changing bootstrap commit. Should do the trick
https://github.com/waku-org/js-waku/pull/1288